sandy/core/cov.py

```diff
@@ -557,57 +565,28 @@ def sampling(self, nsmp, seed=None):
         Draw 3 sets of samples using custom seed.
         >>> sandy.CategoryCov([[1, 0.4],[0.4, 1]]).sampling(3, seed=11)
                      0            1
-        0 -1.74945e+00 -3.13159e+00
-        1  2.86073e-01  1.06836e-01
-        2  4.84565e-01 -9.91209e-02
+        0  1.74945e+00 -1.73202e+00
+        1 -2.86073e-01 -1.22022e-01
+        2 -4.84565e-01 -4.86773e-01
+
+        >>> sandy.CategoryCov([[1, 0.4],[0.4, 1]]).sampling(3, seed=11, rows=1)
+                     0            1
+        0  1.74945e+00 -1.73202e+00
+        1 -2.86073e-01 -1.22022e-01
+        2 -4.84565e-01 -4.86773e-01

```

Comment:
To prove that these samples are correct, can you add a test when you sample from the same 2×2 covariance matrix (same seed). This time take 1 million samples. The resulting covariance (calculated with no.cov) should be equal to the original one within a tolerance limit. Also, the mean of the samples should be 0 within a tolerance limit.

Reply:
@AitorBengoechea , this should be kept in mind